src/components/exchange/ConfirmExchangeButton.js

```diff
@@ -57,6 +60,20 @@ export default function ConfirmExchangeButton({
     !isSufficientGas ||
     !isSufficientLiquidity;
 
+  const shadowsForAsset = useMemo(
+    () => [
+      [0, 10, 30, colors.dark, 0.2],
+      [0, 5, 15, colorForAsset, 0.4],
+    ],
+    [colorForAsset]
+  );
+
+  const isRouteSwapDetails = useMemo(
+    () => routeName === Routes.SWAP_DETAILS_SHEET,
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
```

Comment:
> [](url)
```suggestion
    [routeName]
```

I see no point in excluding this from dep array

Reply:
it's so that the ConfirmExchangeButton shadow in the swap modal doesn't change when you open the swap details modal...but maybe there's a better way to prevent it from rerendering when route changes